.github/workflows/_docs.yml

```diff
@@ -95,7 +99,10 @@ jobs:
         timeout-minutes: ${{ matrix.timeout-minutes }}
         id: build-docs
         env:
-          WITH_PUSH: ${{ github.event_name == 'schedule' || startsWith(github.event.ref, 'refs/tags/v') }}
```

Comment:
What was the old logic here trying to do? It seems like it's only trying to push a PR when a scheduled (nightly) job is running or when a release branch (with the v tag) is running

The new behavior seeeeems to just push docs on nightlies. Have we lost the push on release branches or does that still happen?

Reply:
Hmm, why do we want to change that? I.e. this workflow is run periodically there are no harm in pushing the update, is there?

Reply:
I think the decision to push the doc or not should depend only on the `inputs.push` parameter.  This change actually makes the workflow consistent with the logic on https://github.com/pytorch/pytorch/blob/master/.github/workflows/_docs.yml#L85. The necessary credential to push doc will only be available when `inputs.push` is set to true.

So pushing to a release branch or running pull as a periodic job can both publish the doc as long as the upstream workflow set the `inputs.push` param to true accordingly.